src/main/java/mate/academy/springboot/swagger/repository/ProductRepository.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.springboot.swagger.repository;
+
+import java.math.BigDecimal;
+import mate.academy.springboot.swagger.model.Product;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.repository.PagingAndSortingRepository;
+
+public interface ProductRepository extends PagingAndSortingRepository<Product, Long>,
```

Comment:
`JpaRepository` source code:
```java
public interface JpaRepository<T, ID extends Serializable> extends PagingAndSortingRepository<T, ID> { 
    ...
}
```
so we do not need to also extend `PagingAndSortingRepository`